### ISSUE TITLE
Does this work with Circle Pad Pro?

### ISSUE THREAD

**Issue by fpuente97** [Roberto-Nessy/OoT3D_Standalone_Free_Cam#4]
Hi, so... I tried to use this with the CPP on my O3DS, I now the mods is installed because I can access the camera adjustment menu, but can't manage to make it work.

Nice work though! I heard from people how amazing is this!

**Comment by etillxd** [comment]
Support for the CPP would be great!
Majoras Mask also natively supports it, so it may be possible?
Anyways, if this will ever be worked on, i'd be willing to test it or help in any way that's possible to me.

**Comment by EthanMac1915** [comment]
Yeah if a CPP version was ever going i'd love to help by testing!!
